draft-ietf-rats-architecture.md

```diff
@@ -133,6 +123,10 @@ Attester:
 
 : An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
 
+Claim:
+: A piece of asserted information, often in the form of a name/value pair.
```

Comment:
The word 'asserted' is ambiguous / dissonant - leading the reader to wonder if this is a loaded word - but without context or resolution. I also don't know if by saying to 'compare' to JWT use of 'claim' whether this spec uses it synonymously or if the reader is supposed to look for and internalize how they differ.

Being explicit is better IMO. "A piece of information, often in the form of a name/value pair, that can be asserted as Evidence, Endorsements or Attestation Results."